include/yaml-cpp/node/detail/impl.h

```diff
@@ -129,23 +162,29 @@ inline node& node_data::get(const Key& key, shared_memory_holder pMemory) {
 
 template <typename Key>
 inline bool node_data::remove(const Key& key, shared_memory_holder pMemory) {
-  if (m_type != NodeType::Map)
-    return false;
-
-  for (kv_pairs::iterator it = m_undefinedPairs.begin();
-       it != m_undefinedPairs.end();) {
-    kv_pairs::iterator jt = std::next(it);
-    if (it->first->equals(key, pMemory))
-      m_undefinedPairs.erase(it);
-    it = jt;
-  }
-
-  for (node_map::iterator it = m_map.begin(); it != m_map.end(); ++it) {
-    if (it->first->equals(key, pMemory)) {
-      m_map.erase(it);
-      return true;
-    }
-  }
+	if (m_type == NodeType::Sequence) {
+		char result = remove_idx<Key>::remove(m_sequence, key);
+		if (result == 0) return false;
```

Comment:
Add curly braces around all "if" statements, even if they're one line.

(The codebase isn't quite consistent there, but I'm trying to enforce it for new code.)